scripts/aws.sh

```diff
@@ -0,0 +1,488 @@
+        Fn::Base64: !Sub |
+          #!/bin/bash -xe
+          echo "====== Authorizing public key ======" | tee -a /tmp/init_output.txt
+          echo "\${PublicKeyString}" >> /home/ec2-user/.ssh/authorized_keys
+          # Use the same defaults as OCP to avoid failing requests to apiserver, such as
+          # requesting logs.
+          echo "====== Updating inotify =====" | tee -a /tmp/init_output.txt
+          echo "fs.inotify.max_user_watches = 65536" >> /etc/sysctl.conf
+          echo "fs.inotify.max_user_instances = 8192" >> /etc/sysctl.conf
+          sysctl --system |& tee -a /tmp/init_output.txt
+          sysctl -a |& tee -a /tmp/init_output.txt
+
+Outputs:
+  InstanceId:
+    Description: RHEL Host Instance ID
+    Value: !Ref RHELInstance
+  PrivateIp:
+    Description: The bastion host Private DNS, will be used for cluster install pulling release image
+    Value: !GetAtt RHELInstance.PrivateIp
+  PublicIp:
+    Description: The bastion host Public IP, will be used for registering minIO server DNS
+    Value: !GetAtt RHELInstance.PublicIp
+EOF
+
+  aws --region "${region}" cloudformation create-stack --stack-name "${stack_name}" \
+    --template-body "file://${cf_tpl_file}" \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --parameters \
+    ParameterKey=HostInstanceType,ParameterValue="${inst_type}" \
+    ParameterKey=Machinename,ParameterValue="${stack_name}" \
+    ParameterKey=AmiId,ParameterValue="${ami_id}" \
+    ParameterKey=EC2Type,ParameterValue="${ec2Type}" \
+    ParameterKey=PublicKeyString,ParameterValue="$(cat "${pub_key}")"
+    
+  #echo "Waiting for stack to be created"
+  aws --region "${region}" cloudformation wait stack-create-complete --stack-name "${stack_name}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack to be created" "Stack created successfully"
+
+  # shellcheck disable=SC2016
+  instance_id="$(aws --region "${region}" cloudformation describe-stacks --stack-name "${stack_name}" \
+    --query 'Stacks[].Outputs[?OutputKey == `InstanceId`].OutputValue' --output text)"
+
+  aws --region "${region}" ec2 wait instance-status-ok --instance-id "${instance_id}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack status to be OK" "Stack status OK"
+
+  # shellcheck disable=SC2016
+  public_ip=$(aws --region "${region}" cloudformation describe-stacks --stack-name "${stack_name}" \
+    --query 'Stacks[].Outputs[?OutputKey == `PublicIp`].OutputValue' --output text)
+
+  rm -f "${cf_tpl_file}"
+  
+  echo "PUBLIC IP: ${public_ip}"
+}
+
+action_delete() {
+  aws --region "${region}" cloudformation delete-stack --stack-name "${stack_name}"
+	aws --region "${region}" cloudformation wait stack-delete-complete --stack-name "${stack_name}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack to delete" "Stack deleted"
+	exit 0
+}
+
+action_describe() {
+  aws --region "${region}" cloudformation describe-stack-events --stack-name "${stack_name}" --output json | less
```

Comment:
Let's leave it to the script user if to paginate the output or not?
```suggestion
  aws --region "${region}" cloudformation describe-stack-events --stack-name "${stack_name}" --output json
```

Reply:
It's because of the `export AWS_PAGER=""` - empty value means no pager hence the `| less` redirection.

Reply:
Removed the `export AWS_PAGER=""` and the `less` command